index.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+    <script type="text/javaScript" src="js/main.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" onclick="showNav()">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="mobile-nav">
+        <i class="fas fa-times" id="close-btn" onClick="hideNav()"></i>
+        <ul>
+          <li><a href="#projects" onClick="hideNav()">Portoflio</a></li>
+          <li><a href="#about-me" onClick="hideNav()">About</a></li>
+          <li><a href="#contact" onClick="hideNav()">Contact</a></li>
+        </ul>
+        <img id="shape-1" src="media/svg/square.svg" />
+        <img id="shape-2" src="media/svg/circle.svg" />
+        <img id="shape-3" src="media/svg/rectangle.svg" />
```

Comment:
- [ ] Good job on adding the onClick event 👍 However when updating the required changes from the previous CR you also added images to your mobile menu, by adding these images you are not following the style requirements anymore for this project. Another issue this raises is that the image in line 38 is overlapping your link in line 34 and therefore the user is not able to use it anymore.